tools/SecurityTools/RunBinSkim.ps1

```diff
@@ -0,0 +1,10 @@
+$repoRoot = "$PSScriptRoot/../.."
+$binSkim = (Get-Item -Path "$repoRoot/tools/SecurityTools/Microsoft.CodeAnalysis.BinSkim*/tools/*/win-x64/BinSkim.exe" | Select-Object -First 1).FullName
+$dllBaseFolder = "$repoRoot/artifacts/Debug"
```

Comment:
Do we only run security for debug configuration?

Reply:
in "azure-powershell-security-tools": 
[https://dev.azure.com/azure-sdk/internal/_build?definitionId=248&_a=summary](url)

It runs for debug configuration only; build arguments:
"build.proj /t:Build;GenerateHelp"